doc/tools/coqrst/coqdomain.py

```diff
@@ -1093,7 +1097,10 @@ def add_coqtop_output(self):
         """Add coqtop's responses to a Sphinx AST
 
         Finds nodes to process using is_coqtop_block."""
-        with CoqTop(color=True) as repl:
+        additional_args = []
+        for node in self.document.traverse(CoqtopBlocksTransform.is_SPHINXCOQEXTRAFLAGS_field):
+            additional_args.extend((node.children[1].rawsource).split())
```

Comment:
LGTM, but:

- I suggest changing the name to something like `coqtop_args`, since this is not specific to Sphinx
- Using `shlex.split` to do slightly smarter splitting

With these two things taken together, I'd write something like this:

```
arg_fields = self.document.traverse(CoqtopBlocksTransform.is_coqtop_args_field)
additional_args = [arg for field in arg_fields for arg in shlex.split(field.children[1].rawsource)]
```